workshops/views.py

```diff
@@ -707,6 +716,28 @@ def person_edit(request, person_id):
                 messages.error(request, 'Fix errors in the award form.',
                                extra_tags='awards')
 
+        elif 'certificate-badge' in request.POST:
+            certificate_form = PersonCertificateForm(request.POST, prefix='certificate')
+
+            if certificate_form.is_valid():
+                certificate = certificate_form.save()
+
+                messages.success(
+                    request,
+                    '{person} was awarded a {badge} certificate.'.format(
+                        person=str(person),
+                        badge=certificate.badge.title,
+                    ),
+                    extra_tags='certificates',
```

Comment:
This is a trick used by other tabs ("Awards", "Tasks") so that when e.g. PersonAwardForm is submitted and new award is added, the page switches back to "Awards" tab.

For some reason JS doesn't work with "certificates" tab.